homeassistant/components/overkiz/__init__.py

```diff
@@ -164,11 +167,7 @@ async def async_setup_entry(hass: HomeAssistant, entry: ConfigEntry) -> bool:
 
 async def async_unload_entry(hass: HomeAssistant, entry: ConfigEntry) -> bool:
```

Comment:
Let's also use it here

Reply:
Good one.. Fixed in 6328c2e285d6d24fc1acf4fccfcee4d57693cbcb.